stonesoup/orbital_functions.py

```diff
@@ -0,0 +1,415 @@
+# -*- coding: utf-8 -*-
```

Comment:
May be we should create a functions submodule. Can move content of `functions.py` to `functions/__init__.py` initially, and then this file can be `functions/orbital.py`.

Reply:
Sounds sensible. But also sounds sensible to have `functions/trigonometric.py`, `functions/coordinate_transforms.py`, `etc/etc.py`. Do that here, or in a different PR?

Reply:
I agree, but probably do that on another change, as it will effect loads more files.

Reply:
> > > Haven't reviewed documentation yet, as the new modules also need inclusion into documentation (with `automodule`) in order to build.
> > 
> > 
> > You mean I add an instance of automodule: to stonesoup.types.rst covering the OrbitalState class and its children? Or is this done automatically elsewhere?
> 
> Yeah, and extra file/links for orbital functions.

ok - see new docs. I've also attempted to place them somewhere clever, so `OrbitalState` is a subsection of `State` and "orbital functions" a subsection of "functions". Check that makes sense.